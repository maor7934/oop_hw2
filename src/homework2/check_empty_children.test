CreateNode n1 1
CreateGraph A
AddNode A n1
ListChildren A n1