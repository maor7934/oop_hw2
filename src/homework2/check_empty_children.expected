created node n1 with cost 1
created graph A
added node n1 to A
the children of n1 in A are:
